software/database/pom.xml

```diff
@@ -107,18 +107,10 @@
             <artifactId>groovy-all</artifactId>
         </dependency>
         <dependency>
-            <groupId>com.beust</groupId>
-            <artifactId>jcommander</artifactId>
-        </dependency>
-        <dependency>
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
         </dependency>
         <dependency>
-            <groupId>com.google.code.findbugs</groupId>
-            <artifactId>jsr305</artifactId>
```

Comment:
I prefer to see explicit dependencies rather than transitive dependencies. It makes intent clearer, it means there's no surprises if the transitive dependencies change, and it reduces the noise from `mvn dependency:analyze`.

In the past I did go through and make all dependencies explicit - it made the initial process of dependency license discovery with `mvn dependency:analyze` much easier back when we were doing due diligence ahead of joining the incubator. I've not been following it up with further checks, though, so it probably has slipped.

Other transitive dependencies supply `@Nullable` however - jclouds has an implementation - so perhaps `DatastoreMixins` is referring to a non-JSR305 `@Nullable`? If so that is where the fix should be made.